tests/test_service.py

```diff
@@ -116,3 +136,20 @@ def test_update_in_stock(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         updated_inventory_item = resp.get_json()
         self.assertEqual(updated_inventory_item["in_stock"], False)
+
+    def test_delete_inventory_item(self):
+        """Delete an inventory item"""
+        # Create a dummy inventory item
+        test_inventory_item = self._create_inventory_items(1)[0]
+
+        # Send a request to delete the dummy item
+        resp = self.app.delete(
+            "{0}/{1}".format(BASE_URL, test_inventory_item.id),
+            content_type=CONTENT_TYPE_JSON,
+        )
+
+        # Ensure the response code is 204 and contains no data
+        self.assertEqual(resp.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(len(resp.data), 0)
+
+        # TODO: Once the query endpoint is ready, verify that the pet is actually gone
```

Comment:
To make this more complete, we should also assert that the item was indeed deleted once the `QUERY` has landed.